src/Calculator/Resources/am-ET/Resources.resw

```diff
@@ -901,10 +901,7 @@
     <value>የግራ ቅንፍ</value>
     <comment>Screen reader prompt for the Calculator "(" button on the scientific operator keypad</comment>
   </data>
-  <data name="Format_OpenParenthesisAutomationNamePrefix" xml:space="preserve">
```

Comment:
You shouldn't need to remove this for every language, just the en-us resw, the rest should be removed automatically by our tooling if I am not mistaken. @mcooley can you double check me here?

Reply:
@joseartrivera yep, any changes in the non-English resw files will just get overwritten by the localization tools. But there's no real harm in modifying the files if you need to validate changes.

Reply:
I prefer to remove all of them (via a regex, it takes only 10 seconds) so I can test with other languages (in fact, I randomly dev with the app in French, Spanish or one of the RtL language) to be sure to be sure the app is fully localized and support all types of region settings